virtual_ecosystem/models/animal/animal_cohorts.py

```diff
@@ -134,13 +139,25 @@ def excrete(self, excreta_mass: float, excrement_pool: DecayPool) -> None:
 
         Args:
             excreta_mass: The total mass of carbonaceous wastes excreted by the cohort.
-            excrement_pool: The pool of wastes to which the excreted nitrogenous wastes
+            excrement_pools: The pools of waste to which the excreted nitrogenous wastes
                 flow.
 
         """
-        excrement_pool.decomposed_energy += (
-            excreta_mass * self.constants.nitrogen_excreta_proportion
-        )
+        # the number of communities over which the feces are to be distributed
+        number_communities = len(excrement_pools)
+
+        excreta_mass_per_community = (
+            excreta_mass / number_communities
+        ) * self.constants.nitrogen_excreta_proportion
```

Comment:
An this basically means that a cohort distributes its excrement evenly over the grid cells than comprise its territory?

Reply:
That's right. Current assumptions have waste deposited evenly across a cohort's territory.